src/utilities/types/CartItemType.ts

```diff
@@ -0,0 +1,9 @@
+import { Product } from "./ProductType"
+
+export type CartItem = Product & {
```

Comment:
Ja bym nie przechowywał w Carcie tych wszystkich danych o produkcie.
IMO wystarczy ID produktu, rozmiar i jego ilość. Resztą można pobrać z `ProductContext` robiąc finda po ID. Dzięki temu te dwa elementy systemu będą bardziej czyste i oddzielone od siebie. Cart nie musi posiadać informacji wszystkich o danym produkcie, niech przechowuje tylko to co musi.

Reply:
{
userId: 43
firstName: 'test'
}